turbo-txpool/src/lib.rs

```diff
@@ -87,3 +228,138 @@ impl server::Txpool for TxPool {
         Box::pin(self.get_transactions(request))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    fn inner() -> Inner {
+        let cfg = Config::builder().max_txs(10).build();
+        Inner::with_config(cfg)
+    }
+
+    fn tx(gas_price: u64) -> Tx {
+        Tx {
```

Comment:
Maybe slightly cleaner to instantiate this as `Tx::default()` then modify the values accordingly?

Reply:
Does `Tx` having a `Default` implementation make sense?

Reply:
No, not *really* + I for some reason I misread the `Debug` derivation as `Default`. So it's probably fine to ignore this.